src/Middleware/RequestThrottling/perf/Microbenchmarks/QueueFullOverhead.cs

```diff
@@ -11,19 +11,26 @@ namespace Microsoft.AspNetCore.RequestThrottling.Microbenchmarks
 {
     public class QueueFullOverhead
     {
-        private const int _numRequests = 2000;
+        private const int _numRequests = 200;
         private int _requestCount = 0;
         private ManualResetEventSlim _mres = new ManualResetEventSlim();
 
-        private RequestThrottlingMiddleware _middleware;
+        private RequestThrottlingMiddleware _middleware_FIFO;
+        private RequestThrottlingMiddleware _middleware_LIFO;
 
         [Params(8)]
         public int MaxConcurrentRequests;
 
         [GlobalSetup]
         public void GlobalSetup()
         {
-            _middleware = TestUtils.CreateTestMiddleware_TailDrop(
+            _middleware_FIFO = TestUtils.CreateTestMiddleware_TailDrop(
+                maxConcurrentRequests: MaxConcurrentRequests,
+                requestQueueLimit: _numRequests,
+                next: IncrementAndCheck
+                );
```

Comment:
I don't care a whole lot, but this is pretty non-standard for our codebase. Normally we'd just put this on the end of the previous line.